src/Sdk/Coverage/types.ts

```diff
@@ -14,9 +14,7 @@ export interface CoverageUpdateRequest {
     url?: string | null;
 }
 
-export interface CoverageCreateRequest extends CoverageUpdateRequest {
-    author: Contact['id'] | string;
-}
+export interface CoverageCreateRequest extends CoverageUpdateRequest {}
```

Comment:
Hijacked this PR to revert the incorrect type fix https://github.com/prezly/javascript-sdk/commit/0e66a7e6774566a76b9ca3da81302f6816a28e4f.
Mentioned by @e1himself here https://prezly.slack.com/archives/C029LTE0M/p1574349891215900?thread_ts=1574344190.213000&cid=C029LTE0M

Reply:
The type is now updated to match the latest type in the backend app (7678d4259f7025bff2b49e9a99597588a52615c9).